golem-worker-service/src/api/worker.rs

```diff
@@ -314,6 +314,31 @@ impl WorkerApi {
 
         Ok(Json(ResumeResponse {}))
     }
+
+    #[oai(
+        path = "/:component_id/workers/:worker_name/update",
+        method = "post",
```

Comment:
should not it be `PUT` with  path `/:component_id/workers/:worker_name` ?

Reply:
I'm not sure, because it is not really an update in itself - it just registers an update attempt that happens asynchronously and may fail (and this failure is not reported by this request's response). So for me it is more like the pause, resume, etc actions on a worker.

Reply:
`POST` = create (id not specified in URL)
`PUT` = update (id specified in URL)